src/index.spec.js

```diff
@@ -1,70 +1,211 @@
+/* eslint-env mocha */
+/* eslint-disable
+  no-unused-expressions,
+  react/jsx-filename-extension,
+  import/no-extraneous-dependencies
+*/
+
 import React from 'react'
 import { mount } from 'enzyme'
 import { expect, spy } from 'chai'
-
+import blanket from 'blanket' // eslint-disable-line
 import loader from './index'
 import Dots from './Dots'
 
+
 const Component = () => <div />
 const Loader = () => <div />
 const getWrapped = (config, props) => {
   const Container = loader(Component, config)
   return mount(<Container {...props} />)
 }
 
-/* eslint-env mocha */
-/* eslint-disable no-unused-expressions */
 
 describe('react-loader', () => {
-  describe('default behaviour', () => {
-    it('should waits', () => {
-      const loaded = getWrapped()
-      expect(loaded.find(Component).node).to.be.undefined
-    })
-
-    it('should calls the load function once', () => {
-      const load = spy(() => {})
-      const loaded = getWrapped(undefined, { load })
-      loaded.setProps({ loaded: true })
-      load.should.have.been.called.once()
-    })
-
-    it('should shows a Dots component when waiting', () => {
-      const loaded = getWrapped()
-      loaded.find(Dots).node.should.exists
-    })
-
-    it('should shows the component when the Boolean(prop) is true', () => {
-      const loaded = getWrapped()
-      loaded.setProps({ loaded: true })
-      loaded.find(Component).node.should.exists
-    })
-
-    it('should keeps props', () => {
-      const loaded = getWrapped({ Loader }, { test: 'prop' })
-      loaded.find(Loader).props().should.be.deep.equals({ test: 'prop' })
-      loaded.setProps({ test: 'prop2', loaded: true })
-      loaded.find(Component).props().should.be.deep.equals({ test: 'prop2', loaded: true })
-    })
+  it('should wait for a `data` props [readme]', () => {
+    // Mount
+    const load = spy(() => {})
+    const loaded = getWrapped({ wait: ['data'] }, { load })
+
+    // Load function is called
+    // Graphic component isn't called
+    // Loader should be Dots
+    load.should.have.been.called.once()
+    expect(loaded.find(Component).node).to.be.undefined
+    loaded.find(Dots).node.should.exists
+
+    // Change `data` value
+    loaded.setProps({ data: true })
+
+    // Load function is not called twice
+    // Graphic component is called
+    // Loader shouldn't be printed
+    load.should.have.been.called.once()
```

Comment:
You don't have to call twice the line called.once(). Parenthesis are [unnecessary](http://chaijs.com/plugins/chai-spies/#once)